docs/releasing.md

```diff
@@ -61,18 +61,6 @@ cd ../lighthouse-pristine
 yarn
 yarn build-all
```

Comment:
`preparer-pristine.sh`, `yarn` and `yarn build-all` are all run in `prepare-package.sh`

I think this entire block is not needed anymore.